packages/expo-cli/lib/configure-plugin.js

```diff
@@ -1,9 +1,38 @@
 const { join } = require('path')
 const { readFile, writeFile } = require('fs')
 const { promisify } = require('util')
+const { spawn } = require('child_process')
+const { blue } = require('kleur')
 
 const plugin = '@bugsnag/plugin-expo-eas-sourcemaps'
 
+function usingYarnV2 (projectRoot) {
```

Comment:
Not really a big deal but I guess this should be `usingYarnV2AndUp` or something?  At least when we do the manual documentation we should use that kind of phrasing anyway.

Reply:
good shout - or I could flip it on it's head and make it `usingYarnClassic`